index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+        <link rel="stylesheet" href="./style.css">
+        <title>Calculate with Jake</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="">
+        <script src="./main.js"></script>
+    </head>
+    <body>
+        <h1 class="center">Calculate with Jake</h1>
+
+        <div class="topCalc">
+            <div class="calcField">
+                <input type="text" class="calcDisplay" id="calc" value=0>
+            </div>
+        </div>
+
+        <script>
```

Comment:
Let's break your js code out into a separate file to keep separation of concerns. that way it will be easier to add to it in the future.

Reply:
Moved to main.js